src/components/stories/Preview.js

```diff
@@ -0,0 +1,149 @@
+import React, { DOM as dom } from "react";
+import { storiesOf, action } from "@kadira/storybook";
+import _Preview from "../Editor/Preview";
+const Preview = React.createFactory(_Preview.WrappedComponent);
```

Comment:
nit: I find this a bit confusing at first(`_Preview` and `Preview`), but I guess we have no other choice than doing something like that, and I can't find a better naming